pkg/etcdsync/etcdsync_suite_test.go

```diff
@@ -17,8 +17,6 @@
 package etcdsync_test
 
 import (
-	_ "github.com/apache/incubator-servicecomb-service-center/server/core/registry/embededetcd"
```

Comment:
should not delete these imports

Reply:
there is no package named `server/core/registry/embededetcd`

Reply:
this seems to be an outdated test code

Reply:
I have updated the code based on your comments